src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -158,7 +158,9 @@ export const Browser: FunctionComponent<BrowserProps> = (
       ) {
         changeSelectedSpecies(activeGenomeId);
       } else {
-        changeSelectedSpecies(props.committedSpecies[0].genome_id);
+        if (props.committedSpecies[0]) {
```

Comment:
committedSpecies have already been destructured from props several lines above; might as well use them directly

Reply:
That's done